projects/widgets/docs/getting-started.md

```diff
@@ -1,3 +1,66 @@
-# getting started
+# Getting started
 
-...
+## 1. Registering a widget project
+
+Before you can start building your widget you must make a widget project on our [self-service platform Projectaanvraag](https://projectaanvraag.uitdatabank.be). The registration process is very simple and only takes a few minutes.
+
+The created widget project will use event data from our test environment. No payment is required until you decide you want to use real event data.
+
+<https://www.youtube.com/watch?v=X0uplBw98gg>
```

Comment:
FYI: the screen capture could be better (e.g. it stops quite abruptly) but since we are building a new Projectaanvraag I didn't invest time in getting the screen capture better